t/deploy.t

```diff
@@ -81,10 +81,6 @@ ok($dh->version_storage->database_version, 'DB deployed');
 is($dh->version_storage->database_version, $dh->schema_version, 'Schema at correct version');
 is($ret, 0, 'Expected return value (0) for no action needed');
 
-# check systemuser exists after upgrade
-my $user = $schema->resultset('Users')->find({username => 'system'});
-ok($user->id, 'system user does exists');
-
```

Comment:
It's actually explained in the current version of the commit message. It *is* related to the commit, though it's not obvious at first - it made me scratch my head for a while till I figured it out.

The schema upgrade test works by deploying the schema at version "current version minus 2", then upgrading it. This particular assertion assumes that the 'system' user creation will happen as part of that upgrade process. However, the creation of the 'system' user *as part of schema upgrade* only occurs when upgrading from schema version 50 or earlier.

So this breaks purely because this commit *bumps the schema version*. As long as the schema version was 52, the schema upgrade test would deploy schema version 50 and upgrade to 52, and that process *would* include the 'system' user creation, and the assertion would pass. However, since this commit bumps the schema version to 53, the upgrade test now deploys schema version 51 and upgrades to version 53. The 'system' user is *not* created as part of that transaction, and the assertion fails.

So I think it does belong in this commit.